src/main/java/com/treasuredata/client/model/TDFederatedQueryConfig.java

```diff
@@ -0,0 +1,165 @@
+package com.treasuredata.client.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+public class TDFederatedQueryConfig
+{
+    private final int id;
+    private final String type;
+    private final int userId;
+    private final int accountId;
+    private final String name;
+    private final int connectionId;
+    private final String createdAt;
+    private final String updatedAt;
+
+    @JsonDeserialize(using = com.treasuredata.client.deserialize.FederatedQueryConfigSettingsDeserializer.class)
+    private final String settings;
```

Comment:
If this is anonymous json serialized, `Map<String, Object>` should be enough

Reply:
Updated like that.